libs/alac/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2016 OpenWrt.org  
```

Comment:
Don't assign copyright to OpenWrt unless you're working under contract or affiliated.